apps/worker/src/app/workflow/usecases/send-message/send-message-push.usecase.ts

```diff
@@ -148,21 +153,26 @@ export class SendMessagePush extends SendMessageBase {
       await this.sendSelectedIntegrationExecution(command.job, integration);
 
       const overrides = command.overrides[integration.providerId] || {};
-
-      const result = await this.sendMessage(
-        subscriber,
-        integration,
-        deviceTokens,
-        title,
-        content,
-        command,
-        command.payload,
-        overrides,
-        stepData
-      );
-
-      if (!result) {
-        integrationsWithErrors++;
+      const target = (overrides as { deviceTokens?: string[] }).deviceTokens || deviceTokens;
```

Comment:
Extracting the `deviceTokens` out of `this.sendMessage` allowed us to easily loop them reusing the new flow implemented in https://github.com/novuhq/novu/pull/4495.